### ISSUE TITLE
Accessibility - Login Page

### ISSUE THREAD

**Issue by tander08** [QualitativeDataRepository/dataverse#84]
There are a number of issues that appear only on the login page (https://idp.qdr.syr.edu/idp/profile/SAML2/Redirect/SSO?execution=e3s1) I've listed them below:

## Border of Page 
- [x] html element must have a lang attribute
- [ ] Page must have means to bypass repeated blocks
- [x] Document should have one main landmark
- [x] Page should contain level-one heading

(all above refer to the whole page - see picture):
![image](https://user-images.githubusercontent.com/89411909/134062200-d263159f-0867-422a-afef-208ccf941a1b.png)

- [x] All page content should be contained by landmarks
![image](https://user-images.githubusercontent.com/89411909/134062150-c6f29be5-a953-45f7-9626-ff2225242bf7.png)

## Color Contrast
- [x] Some of the text on the bottom banner is light grey and needs more contrast (About, Terms and Conditions, Policies, Site Map, qdr@syr.edu, CoreTrustSeal Standards, Center for Qualitative and Multi-Method Inquiry, Maxwell School of Citizenship and Public Affairs, Syracuse University)

**Comment by qqmyers** [comment]
I've fixed these on dev/stage. I also fixed the various logout/error views that only appear briefly/only upon error. One thing to check w.r.t. style - adding an H1 heading means I had to put new text on the screen - right now a basic QDR Login/QDR Logout. If that text or the style of it should change, let me know.

**Comment by qqmyers** [comment]
FWIW: the repeated blocks warning above is gone, but I didn't specifically address it - I think other changes resolved it.
